landscape_config.yaml

```diff
@@ -94,33 +94,40 @@ misc:
 charts:
 - name: gardener
 - name: identity
+  # identity is the identity provider for the dashboard
+  # this addon specifies who can login into the dashboard and how
+  # there are basically two options:
+  # - you specify accounts in this file: see "staticPasswords"
+  # - you use a dex connector to use OAuth authentication for the dashboard: see "connectors"
+  # You need at least one entry in one of the categories, otherwise the dashboard won't be accessible!
   values:
-  # You need to set at least one password here to be able to
-  # log on. You can either provide the password in clear text
+  # You can either provide the password in clear text
   # in the "password" key or provide bcrypted passsword in
   # the "hash" key. Bcrypted password can be created with
   # htpasswd -bnBC 10 "" password | tr -d ':\n' | sed 's/$2y/$2a/'
   staticPasswords:
-  # sample
-  #- email: "dirk@**********"
-  #  password: "***************"
-  #  username: "admin"
-  #  userID: "08a8684b-db88-4b73-90a9-3cd1661f5466"
+  # example 
+  - email: "some.mail@example.org"
+    hash: "$2a$10$JIKb36h2oM5pVAvvM/EniemkQJhCtdmmsVamyRmNVJhCyOlTY72uu" # example hash for password "password"
```

Comment:
I would write this like you did elsewhere, and remove the unnecessary comments:

 email: \<email address>\>
 hash: \<bcrypted password\>
 username: \<user name\>
 userID: \<UUID\>